core/application/impl/kagome_application_impl.cpp

```diff
@@ -40,7 +40,7 @@ namespace kagome::application {
     }
 
     Threshold threshold =
-        boost::multiprecision::uint256_t(0.58 * 1e77) / authorities.size();
+        boost::multiprecision::uint128_t(0.58 * 1e77) / authorities.size();
```

Comment:
```suggestion
        boost::multiprecision::uint128_t("102084710076281554150585127412395147264");
```
Lets use this value for now. It is good for testing node as we know that often VRF values are less than this threshold. In future we will change the way we calculate threshold